src/store/modules/gameList.js

```diff
@@ -44,6 +45,7 @@ export default {
       const updatedGame = state.openGames.find((game) => game.id === data.id);
       if (updatedGame) {
         updatedGame.status = data.newStatus;
+        updatedGame.numPlayers = data.numPlayers;
```

Comment:
I'm looking at the `HomeView` call of this function and it doesn't look like it specifies the `numPlayers`. Is the caller of this mutation now expected to provide that?

Reply:
I don't think this is needed actually